Src/ReleaseHistory.md

```diff
@@ -18,10 +18,13 @@
 
 - FPC: Improving RabbitMQ regex (removing new lines and spaces) from secret.
   [#548](https://github.com/microsoft/sarif-pattern-matcher/pull/548)
-- FPC: Normalizing regular expressions (`\s\n` got replaced by `\s`). Rules 
+- FPC: Normalizing regular expressions (`\s\n` got replaced by `\s`). Rules
   `SEC101/036.MySqlCredential`, `SEC101/037.SqlCredentials`,
   `SEC101/038.PostgreSqlCredentials` won't accept spaces in `id` and `secret`.
   [#550](https://github.com/microsoft/sarif-pattern-matcher/pull/550)
+- FPC: Improving MongoDB regular expression removing hosts that contain `\s` or
+  `,` characters.
```

Comment:
'Eliminate whitespace and characters from MongoDB host matches (and resulting fingerprints)'.